src/plugins/controls/public/options_list/components/options_list_control.tsx

```diff
@@ -103,7 +110,18 @@ export const OptionsListControl = ({ typeaheadSubject }: { typeaheadSubject: Sub
         numActiveFilters={validSelectionsCount}
         hasActiveFilters={Boolean(validSelectionsCount)}
       >
-        {hasSelections ? selectionDisplayNode : OptionsListStrings.control.getPlaceholder()}
+        {hasSelections ? (
+          <>
+            {negate && (
+              <EuiTextColor color="danger">
+                <b>{OptionsListStrings.control.getNegate()}</b>{' '}
+              </EuiTextColor>
+            )}
```

Comment:
This design is slightly different than the Figma, because we still want to show the field's title rather than replacing it with `not`: